_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -22,19 +22,24 @@ When the primary shard sends a checkpoint to replica shards on a refresh, a new
 
 Segment replication is the first feature in a series of features designed to decouple reads and writes in order to lower compute costs.
 
+**Limitation:** Segment replication is not compatible with the [Alerting]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) plugin. The Alerting plugin relies on the latest data being available on replica shards when using the `immediate` refresh policy. Segment replication does not provide the same guarantee, which means shards can become stale. This especially impacts [document-level monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/api/#document-level-monitors).   
```

Comment:
>Segment replication does not provide the same guarantee, which means shards can become stale.

It is not clear what do you mean by same guarantee. Can you elaborate more on this ?

Reply:
@lezzago @anasalkouz Can you also review this content.

Reply:
@dreamer-89 Revised. Please review for accuracy.

Reply:
Revised